### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by Ricky-Kynd**
if merged adds the landing component and some react routing. Still waiting on components to be connected on router.

### CODE REVIEW

File: client/css/skeleton.css [hunk 1]
```diff
@@ -0,0 +1,418 @@
...
+@media (min-width: 550px) {}
+
+/* Larger than tablet */
+@media (min-width: 750px) {}
+
+/* Larger than desktop */
+@media (min-width: 1000px) {}
+
+/* Larger than Desktop HD */
+@media (min-width: 1200px) {}
```
Comment (josh-jacobsen) [inline client/css/skeleton.css:418]: ideally this would have been seperate PR to the router

File: client/index.js [hunk 2]
```diff
@@ -1,10 +1,13 @@
 import React from 'react'
 import { render } from 'react-dom'
+import { Router, Route, Link, hashHistory, IndexRoute } from 'react-router'
 import { createStore, applyMiddleware, compose } from 'redux'
 import { Provider } from 'react-redux'
 
-import reducer from './reducer'
+import reducer from './reducers/landingReducer'
```
Comment (josh-jacobsen) [inline client/index.js]: this doesn't look right ... we want to use `reducer.js` and either put everyones reducers into that file, or have several files and use combine reducers.

I recommend we just use one reducer file given where we're at at the moment

File: client/index.js [hunk 3]
```diff
@@ -1,10 +1,13 @@
 import React from 'react'
 import { render } from 'react-dom'
+import { Router, Route, Link, hashHistory, IndexRoute } from 'react-router'
```
Comment (josh-jacobsen) [inline client/index.js:3]: Link isn't used in this file, is it needed ?

**Comment by josh-jacobsen** [comment]
On the whole this looks good. 
I think the comment about the reducer is the only thing that needs attention

🍍

File: client/index.js (hunk 2 above)
Comment (Ricky-Kynd) [inline client/index.js]: Thanks Done!

File: client/index.js (hunk 3 above)
Comment (Ricky-Kynd) [inline client/index.js:3]: Took from docs

File: client/css/skeleton.css (hunk 1 above)
Comment (tony-luisi) [inline client/css/skeleton.css:418]: Yes that makes sense, will note that as a learning for next time.

File: client/index.js (hunk 3 above)
Comment (tony-luisi) [inline client/index.js:3]: You are correct, we don't need it, will update.

**Pull request merged**